backend/routers/transcribe_v2.py

```
@@ -317,12 +317,12 @@ async def send_audio_bytes_developer_webhook(data: bytearray):
         # TODO: add a lock, send shorter segments, validate regex.
         webhook_url = get_user_webhook_db(uid, WebhookType.audio_bytes)
         if not webhook_url:
-            print('No developer webhook url')
+            #print('No developer webhook url')
             return
         webhook_url += f'sample_rate={sample_rate}'
         try:
             response = requests.post(webhook_url, data=data, headers={'Content-Type': 'application/octet-stream'})
```

Ruff

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)